marm-mcp-server/marm_mcp_server/endpoints/notebook.py

```
@@ -20,19 +20,12 @@ async def marm_notebook(request: NotebookRequest):
             action=request.action,
             name=request.name,
             data=request.data,
             names=request.names,
             session_name=request.session_name,
         )
-        if result.get("status") == "error":
-            raise HTTPException(status_code=400, detail=result["message"])
-        return result
-    except HTTPException:
-        raise
     except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)